mopro-ffi/src/mopro.udl

```diff
@@ -12,7 +12,7 @@ dictionary SetupResult {
 
 dictionary GenerateProofResult {
   bytes proof;
-  bytes inputs;
+  sequence<string> inputs;
```

Comment:
Do we need this change? If it is a byte string we can just deserialize it higher up?

It might make sense, just seems orthogonal to accepting bigint inputs